docs/appendices/release-notes/unreleased.rst

```diff
@@ -108,3 +108,5 @@ Fixes
 
 - Fixed an issue that led to an error if a user nested multiple table
   functions.
+
+- Added support for the ``split_part`` scalar function
```

Comment:
Please add this to the CHANGES section. Additionally, our pattern is to add new things on top of the list.